apps/src/p5lab/spritelab/commands/worldCommands.js

```diff
@@ -24,13 +24,20 @@ export const commands = {
 
   getTime(unit) {
     if (unit === 'seconds') {
-      return this.getAdjustedWorldTime() || 0;
+      return this.getAdjustedWorldTime() - this.timerResetTime || 0;
     } else if (unit === 'frames') {
-      return this.p5.World.frameCount || 0;
+      return this.p5.World.frameCount - this.timerResetFrames || 0;
     }
     return 0;
   },
 
+  resetTimer() {
+    const currentTime = this.getAdjustedWorldTime();
```

Comment:
you don't need these intermediate variables, you can just assign directly like `this.timerResetTime = this.getAdjustedWorldTime();`

Reply:
Got it, done.